tetrateci/create_istio_release.sh

```diff
@@ -28,10 +28,10 @@ cd ..
 git clone https://github.com/istio/release-builder --branch ${BRANCH}
 
 
-# HACK : the github runner runs out of space sometimes so removing the 21 GB dotnet folder
+# HACK : the github runner runs provides 14 GB free space. (https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources).
 # Temporary thing, we should be moving to a custom runner instead.
-echo "Deleting /usr/share/dotnet to reclaim space"
-[ -d "/usr/share/dotnet" ] && sudo rm -rf /usr/share/dotnet
+echo "Deleting /usr/share/dotnet, /opt/ghc, /usr/local/share/boost to reclaim space"
```

Comment:
shall we use a for loop instead?
currently if `dotnet` dir does not exist, I think we will skip checking `ghc`.

```sh
echo "Deleting some directory to reclaim the disk space."
for folder in ("a" "b" "c"); do
  echo "Deleting $folder"
  [ -d $folder ] && sudo rm -rf $folder
done
```

Reply:
good catch, thanks